README.md

```diff
@@ -309,15 +316,14 @@ $client->companies->create([
 ]);
 
 /**
- * Update a company (Note: This method is an alias to the create method.
- * In practice you can use create to update companies if you wish)
```

Comment:
Even though this is true, I think there's no need to be so specific about the internals to avoid creating confusion 😄